guides/operator/custom-job-stages/index.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: single
+title:  "Custom Job Stages"
+sidebar:
+  nav: guides
+---
+
+Much like [Custom Webhook Stages](/guides/operator/custom-webhook-stages/), Custom Job Stages enable Spinnaker operators to extend Spinnaker without having to maintain a custom build of Orca. For users of Kubernetes or Titus, Custom Job Stages offer a native solution for enhancing Spinnaker with functionality with the power of containers.
```

Comment:
I'd swap these two sentences. I think the lede is that it's a good solution for containers (also maybe the fact that it wraps the Run Job stage?).  But I might be misreading what's important here. At least, though, the fact that it's like custom webhook stages is probaby less important, and might be left out entirely.